mantidimaging/gui/windows/spectrum_viewer/view.py

```diff
@@ -41,6 +41,8 @@ class SpectrumViewerWindowView(BaseMainWindowView):
     bin_size_spinBox: QSpinBox
     bin_step_spinBox: QSpinBox
 
+    spectrumWidget: SpectrumWidget
```

Comment:
Can the local name be in snake_case
`spectrum_widget: SpectrumWidget`

Reply:
done!